examples/mne_inverse_connectivity_spectrum.py

```diff
@@ -66,20 +66,21 @@
 fmin, fmax = 7.5, 40.
 sfreq = raw.info['sfreq']  # the sampling frequency
 
-con, freqs, times, n_epochs, n_tapers = spectral_connectivity(
+con = spectral_connectivity(
     label_ts, method='wpli2_debiased', mode='multitaper', sfreq=sfreq,
     fmin=fmin, fmax=fmax, mt_adaptive=True, n_jobs=1)
+freqs = con.freqs
 
-n_rows, n_cols = con.shape[:2]
+n_rows, n_cols = con.get_data(output='full').shape[:2]
```

Comment:
nice!

Reply:
The API is very nice now! At least in my user experience :p